src/commands/local.ts

```diff
@@ -41,23 +42,27 @@ type ClassProcess = {
   listeningPort: number;
 };
 
-// Function that starts the local environment.
-// It also monitors for changes in the user's code and restarts the environment when changes are detected.
-export async function startLocalEnvironment(options: GenezioLocalOptions) {
-  // eslint-disable-next-line no-constant-condition
-  while (true) {
-    // Read the project configuration everytime because it might change
-    const yamlProjectConfiguration = await getProjectConfiguration();
-    let sdk: SdkGeneratorResponse;
-    let processForClasses;
-    let projectConfiguration;
+type BundlerRestartResponse = {
+  shouldRestartBundling: boolean;
+  bundlerOutput?: LocalBundlerOutput;
+};
+
+type LocalBundlerOutput = {
+  success: boolean;
+  projectConfiguration: ProjectConfiguration;
+  processForClasses: Map<string, ClassProcess>;
+  sdk: SdkGeneratorResponse;
+};
 
+export async function bundlerLocalEnvironment(yamlProjectConfiguration: YamlProjectConfiguration): Promise<BundlerRestartResponse> {
```

Comment:
I would suggest instead of `bundlerLocalEnvironment` -> `prepareLocalEnvironment`.